src/bes/nauru/lims/locales/senaite.patient.pot

```diff
@@ -16,74 +16,74 @@ msgstr ""
 
 #: senaite/patient/browser/controlpanel.py:219
 msgid "Allow to publish samples with a temporary MRN"
-msgstr "Allow to publish samples with a temporary Hospital Number"
+msgstr "Allow to publish samples with a temporary Medical Record Number (MRN)"
 
 #: senaite/patient/browser/controlpanel.py:210
 msgid "Allow to verify samples with a temporary MRN"
-msgstr "Allow to verify samples with a temporary Hospital Number"
+msgstr "Allow to verify samples with a temporary Medical Record Number (MRN)"
 
 #: senaite/patient/browser/controlpanel.py:228
 msgid "Display an alert icon on samples with a temporary MRN"
-msgstr "Display an alert icon on samples with a temporary Hospital Number"
+msgstr "Display an alert icon on samples with a temporary Medical Record Number (MRN)"
 
 #: senaite/patient/browser/controlpanel.py:220
 msgid "If selected, users will be able to publish samples that have a Patient assigned with a temporary Medical Record Number (MRN)."
-msgstr "If selected, users will be able to publish samples that have a Patient assigned with a temporary Hospital Number."
+msgstr "If selected, users will be able to publish samples that have a Patient assigned with a temporary Medical Record Number (MRN)."
 
 #: senaite/patient/browser/controlpanel.py:211
 msgid "If selected, users will be able to verify samples that have a Patient assigned with a temporary Medical Record Number (MRN)."
-msgstr "If selected, users will be able to verify samples that have a Patient assigned with a temporary Hospital Number."
+msgstr "If selected, users will be able to verify samples that have a Patient assigned with a temporary Medical Record Number (MRN)."
 
 #: senaite/patient/adapters/listing.py:59
 #: senaite/patient/content/analysisrequest.py:92
 msgid "MRN"
-msgstr "Hospital Number"
+msgstr "MRN"
 
 #: senaite/patient/browser/controlpanel.py:201
 msgid "Make the MRN field mandatory in the sample registration form and when creating or modifying patients."
-msgstr "Make the Hospital Number field mandatory in the sample registration form and when creating or modifying patients."
+msgstr "Make the Medical Record Number (MRN) field mandatory in the sample registration form and when creating or modifying patients."
 
 #: senaite/patient/browser/patientfolder.py:75
 #: senaite/patient/content/analysisrequest.py:66
 msgid "Medical Record #"
-msgstr "Hospital Number"
+msgstr "Medical Record #"
```

Comment:
Remove this entry, cause the `msgstr` equals to the original string defined in `senaite.patient`: https://github.com/senaite/senaite.patient/blob/a6e91b6601285255b3464e55c1e1808d25a37210/src/senaite/patient/locales/senaite.patient.pot#L257